integration/test/app/kubeconfig/kubeconfig_test.go

```diff
@@ -175,7 +175,7 @@ func TestAppLifecycleUsingKubeconfig(t *testing.T) {
 				},
 				Name:      "chart-operator",
 				Namespace: "giantswarm",
-				Version:   "0.10.5",
+				Version:   "0.10.6",
```

Comment:
Without this e2e tests were failing, actual 2.12.0 tiller while 2.14.3 was expected.